components/tools/OmeroJava/test/integration/ProjectionServiceTest.java

```diff
@@ -235,6 +235,19 @@ public void testSameTimepointInterval() throws Exception {
     }
 
     /**
+     * Test the projection with a restricted but valid timepoint range.
+     *
+     * @throws Exception Thrown if an error occurred.
+     */
+    @Test
+    public void testTimepointIntervalNotFullRange() throws Exception {
+        Pixels pixels = importImage();
+        List<Integer> channels = Arrays.asList(0);
+        projectImage(pixels, 2, 4, 1, 3, 1,
```

Comment:
I will need to think of best way of comparing
Might need to look at combining stuff I did for LUT.
Not the goal of the PR
Thanks for the test